kqueue.c

```diff
@@ -141,6 +145,7 @@ kq_init(struct event_base *base)
 		goto err;
 	kqueueop->events_size = kqueueop->changes_size = NEVENT;
 
+#if defined(__APPLE__) && MAC_OS_X_VERSION_MIN_REQUIRED < MAC_OS_X_VERSION_10_5
```

Comment:
I don't think that we should compile it out, since it is `kq_init()` only, so let's leave 58bd1106e9728108ad8f91751e3bc00ea654d5ef